frontend/angular.json

```diff
@@ -209,6 +209,46 @@
                 }
               ]
             },
+            "local": {
+              "fileReplacements": [
+                {
+                  "replace": "projects/upgrade/src/environments/environment.local.ts",
+                  "with": "projects/upgrade/src/environments/environment.ts"
+                }
+              ],
+              "optimization": true,
```

Comment:
Shouldn't this be reversed? Looks like "replace" is all set to `environment.ts` in other parts.

```
            "prod": {
              "fileReplacements": [
                {
                  "replace": "projects/upgrade/src/environments/environment.ts",
                  "with": "projects/upgrade/src/environments/environment.prod.ts"
                }
              ],
```